salt/modules/zcbuildout.py

```diff
@@ -994,7 +997,7 @@ def buildout(directory='.',
 
         salt '*' buildout.buildout /srv/mybuildout
     '''
-    LOG.info('Running buildout in {0} ({1})'.format(directory, config))
+    LOG.info('Running buildout in {0} ({1})'.format(directory, config))  # future lint: disable=str-format-in-logging
```

Comment:
This can just be converted to a %s substitution, unless there is something I am missing. Also, `str-format-in-logging` is not a "future lint" thing, it's actually already present in the newest salt-pylint, which is installed in Jenkins (but this check is currently just disabled in the lint job). So if you find any legit uses in the future, you can just use a normal `# pylint: disable=str-format-in-logging`, and it will be skipped once we enable the check in the jenkins lint job.